code-challenges.test.js

```diff
@@ -23,34 +23,81 @@ const fibonacciLength2 = 10
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// Input: fibonacciLength1, fibonacciLength2
+// Output: [1, 1, 2, 3, 5, 8], [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// use length to represent the length of the sequence
+// uses a while loop to generate the sequence until the length of the array is less than the specified length.
```

Comment:
This is inaccurate, as the while loop will iterate until the length of the array is *not* less than the specified length